cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -88,6 +88,30 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
             }
           });
         });
+
+        // See https://github.com/bbc/simorgh/pull/4352 for more information.
+        if (service === 'news') {
+          it('plays media when a user clicks play', () => {
+            cy.window().then(win => {
+              const media = getBlockData('video', win.SIMORGH_DATA.pageData);
+              if (media && media.type === 'video') {
+                cy.get(
+                  'div[class^="StyledVideoContainer"] img[class^="StyledImg"]',
+                )
+                  .click()
+                  .should('not.exist')
+                  .then(() => {
+                    cy.get('iframe[class^="StyledIframe"]').then($iframe => {
+                      cy.wrap($iframe.prop('contentWindow'), { timeout: 8000 })
+                        .its('embeddedMedia.playerInstances.mediaPlayer')
```

Comment:
Does this property assume only a single media player inside the iframe? Previously when I've looked at this SMP exposes an array of players, the API may have changed since then though.

Reply:
This doesn't assume there is a single media player inside frame `playerInstances.mediaPlayer` refers to a media player with the ID `#mediaPlayer`. A media player called `#audioPlayer` would have an instance name of `audioPlayer`.

Reply:
Fine, if we control and therefore guarantee a stable ID then 👌